hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestBase.java

```diff
@@ -170,7 +170,7 @@ private boolean checkHealth(String fromContainerName, String hostname, int port)
       TestExecStartResultCallback resultCallback =
           executeCommandStringInDocker(fromContainerName, command, false, true);
       String stderrString = resultCallback.getStderr().toString().trim();
-      if (!stderrString.contains("open")) {
+      if (!stderrString.contains("succeeded")) {
```

Comment:
Output message change.

Reply:
let's extract to a constant?

Reply:
It's only used once so I think it's better for readability inline.